lib/reservation_tracker.rb

```diff
@@ -0,0 +1,171 @@
+require_relative 'lib_file_helper'
+
+class ReservationTracker
+  attr_accessor :all_rooms, :all_reservations
+
+  def initialize()
+    # @all_rooms is a collection of all room instances
+    @all_rooms = room_maker
+    # @all_reservations is a collection of all reservation instances
+    @all_reservations = []
+  end
+
+  def room_maker
+    all_rooms = []
+
+    room_numbers = [*1..20]
+    room_numbers.each do |room_num|
+      new_room = Room.new(room_num)
+      all_rooms << new_room
+    end
+    return all_rooms
+  end
+
+  def show_all_hotel_rooms
+    return @all_rooms
```

Comment:
This method's only responsibility is to have return a value stored in `@all_rooms`... However, because you have `attr_accessor :all_rooms` in this class, you are also essentially making a different method the following method of a different name too:
```ruby
def all_rooms
  return @all_rooms
end
```

You probably only need one or the other -- the method `show_all_hotel_rooms` or an `attr_reader` on `:all_rooms`, so I would pick one!